rdr_service/gcloud_functions/retention_eligible_import_function/main.py

```diff
@@ -0,0 +1,89 @@
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+import logging
+
+from aou_cloud.services.gcp_cloud_function import GCPCloudFunctionContext, \
+    FunctionStoragePubSubHandler
+from aou_cloud.services.gcp_cloud_tasks import GCPCloudTask
+
+
+# Function name must contain only lower case Latin letters, digits or underscore. It must
+# start with letter, must not end with a hyphen, and must be at most 63 characters long.
+# There must be a python function in this file with the same name as the entry point.
+function_name = 'retention_eligible_import_function'
+
+# [--trigger-bucket=TRIGGER_BUCKET | --trigger-http | --trigger-topic=TRIGGER_TOPIC |
+# --trigger-event=EVENT_TYPE --trigger-resource=RESOURCE]
+# NOTE: Default function timeout limit is 60s, maximum can be 540s.
+deploy_args = [
+    '--trigger-resource=%%CLOUD_RESOURCE%%',
+    '--trigger-event google.storage.object.finalize',
+    '--timeout=540',
+    '--memory=512'
+]
+
+task_queue = 'resource-tasks'
+_logger = logging.getLogger('function')
+
+
+class RetentionEligibleImportFunction(FunctionStoragePubSubHandler):
+
+    def created(self):
+        """ Handle storage object created event. """
+        _logger.info(f"file found: {self.event.name}")
+
+        cloud_file_path = f'{self.event.bucket}/{self.event.name}'
+
+        data = {
+            "file_path": cloud_file_path,
+            "bucket_name": self.event.bucket,
+            "upload_date": self.event.timeCreated,
+        }
+
+        _logger.info("Pushing cloud task...")
```

Comment:
Optional:  Would it be useful to add details from the data dict to the log message to indicate which file the task is expected to process?